README.md

```diff
@@ -16,52 +16,53 @@ To add the driver to your project, add the following Maven dependency:
 </dependency>
 ```
 
-The class name for this driver is:
+Replace `${jdbc.version}` with the actual version of the driver. It's good practice to manage this version in a properties section of your pom.xml.
 
-```
-com.salesforce.datacloud.jdbc.DataCloudJDBCDriver
-```
+### Driver Class
 
-## Building the driver:
+The driver class name is: `com.salesforce.datacloud.jdbc.DataCloudJDBCDriver`
```

Comment:
what do I need the class name for? Afaict, I don't even need to know this information for using the JDBC driver?